frameit/lib/frameit/editor.rb

```diff
@@ -488,16 +489,9 @@ def build_text_images(max_width, max_height, stack_title)
       results
     end
 
-    # Loads the config (colors, background, texts, etc.)
     # Don't use this method to access the actual text and use `fetch_texts` instead
     def fetch_config
```

Comment:
Are we still using this method? It simply returns `@config` property now, so maybe we could just expose a public getter of it instead of having an extra method?

Reply:
You're right, I will refactor all `fetch_config` to `@config`.

Reply:
[Removed the function in the new commit.](https://github.com/milhauscz/fastlane/commit/91927056a98b8def508b2da405020c105676dfab)